apps/web/src/components/Settings/ManageEmailSection/ManageEmailSection.tsx

```diff
@@ -20,112 +19,76 @@ type Props = {
   queryRef: ManageEmailSectionFragment$key;
 };
 
-const DISABLED_TOGGLE_BY_EMAIL_STATUS = ['Unverified', 'Failed'];
+const SettingsRowTitle = styled(BaseM)`
+  max-width: 380px;
+`;
 
 export default function ManageEmailSection({ queryRef }: Props) {
   const query = useFragment(
     graphql`
       fragment ManageEmailSectionFragment on Query {
         viewer {
           ... on Viewer {
+            user {
+              roles
+            }
             email {
               email
               verificationStatus
-              emailNotificationSettings {
-                unsubscribedFromNotifications
-                unsubscribedFromDigest
-                unsubscribedFromAll
-              }
             }
           }
         }
 
         ...EmailManagerFragment
+        ...useUpdateEmailNotificationSettingsFragment
       }
     `,
     queryRef
   );
 
   const userEmail = query?.viewer?.email?.email;
 
-  const updateEmailNotificationSettings = useUpdateEmailNotificationSettings();
-
-  const currentEmailNotificationSettings = query?.viewer?.email?.emailNotificationSettings;
-
-  const isEmailNotificationSubscribed =
-    !currentEmailNotificationSettings?.unsubscribedFromNotifications;
+  const hasEarlyAccess = useMemo(() => {
+    return query.viewer?.user?.roles?.includes('EARLY_ACCESS');
+  }, [query]);
 
-  const [isEmailNotificationChecked, setIsEmailNotificationChecked] = useState<boolean>(
-    isEmailNotificationSubscribed
+  const emailNotificationSettingData = useMemo(
+    () => [
+      {
+        key: 'notifications',
+        title: 'Notifications',
+        description: 'Notification summary emails',
+      },
+      {
+        key: 'marketing',
+        title: 'General Marketing',
+        description: 'Product marketing emails',
+      },
+      // Conditionally add the 'Members Club' entry if `hasEarlyAccess` is true
+      ...(hasEarlyAccess
+        ? [
+            {
+              key: 'membersClub',
+              title: 'Members Club',
+              description: 'Exclusively for Members Club Holders',
+            },
+          ]
+        : []),
+      {
+        key: 'digest',
+        title: 'Weekly Digest',
+        description: 'Featured Galleries, Collections and more',
+      },
+    ],
+    [hasEarlyAccess]
```

Comment:
1. is there a reason this is copy is distinct from mobile? e.g. marketing email description is `'Product updates, artist collabs, and airdrops'` whereas on web it's just `'Product marketing emails'`
2. we should consolidate this into the hook as well :) and since the hook receives `hasEarlyAccess` bool, we'll be able to return the relevant settings dynamically

Reply:
I consolidated the hooks and regarding the copy, the designs had a different one for mobile and the one for web was from this other screenshot in your loom. Should I bring this up on the slack thread maybe?